src/pool/srv_pool.c

```diff
@@ -2322,6 +2484,10 @@ ds_pool_query_handler(crt_rpc_t *rpc)
 	daos_prop_free(prop);
 }
 
+/**
+ * List a pool's containers
+ */
+
```

Comment:
Forgot to remove?

Reply:
It was accidentally left behind in a manual merge - removed now.